metricbeat/module/mongodb/_meta/kibana/6/dashboard/Metricbeat-mongodb-overview.json

```diff
@@ -37,7 +37,7 @@
                             "enabled": true, 
                             "id": "2", 
                             "params": {
-                                "field": "metricset.host", 
+                                "field": "host.name", 
```

Comment:
`metricset.host` and `host.name` are not the same thing. `host.name` is then name from the host on which the beat runs, `metricset.host` is what address of the service it connects to.

These fields should not have changed in the PR your linked. Same below.

Reply:
Ok, I have reverted both ocurrences of `host.name` to their original values `metricset.host`